src/Sylius/Component/Locale/spec/Model/LocaleSpec.php

```diff
@@ -41,15 +61,56 @@ function its_code_is_mutable()
         $this->getCode()->shouldReturn('de_DE');
     }
 
+    function it_has_a_name()
+    {
+        $this->setCode('pl_PL');
+        $this->getName()->shouldReturn('Polish (Poland)');
+        $this->getName('es')->shouldReturn('polaco (Polonia)');
+
+        $this->setCode('pl');
+        $this->getName()->shouldReturn('Polish');
+        $this->getName('es')->shouldReturn('polaco');
+    }
+
+    function it_returns_name_when_converted_to_string()
+    {
+        $this->setCode('pl_PL');
+        $this->__toString()->shouldReturn('Polish (Poland)');
+
+        $this->setCode('pl');
+        $this->__toString()->shouldReturn('Polish');
+    }
+
     function it_is_enabled_by_default()
     {
-        $this->shouldBeEnabled();
+        $this->isEnabled()->shouldReturn(true);
     }
 
     function it_can_be_disabled()
     {
+        $this->disable();
+        $this->isEnabled()->shouldReturn(false);
+    }
+
+    function it_can_be_enabled()
+    {
+        $this->disable();
+        $this->isEnabled()->shouldReturn(false);
+
+        $this->enable();
+        $this->isEnabled()->shouldReturn(true);
+    }
+
+    function it_can_set_enabled_value()
+    {
+        $this->setEnabled(false);
+        $this->isEnabled()->shouldReturn(false);
+
+        $this->setEnabled(true);
+        $this->isEnabled()->shouldReturn(true);
+
         $this->setEnabled(false);
-        $this->shouldNotBeEnabled();
```

Comment:
Aaaah, I know, I wrote new assertions on `isEnabled` and I used `isEnabled()->shouldReturn(bool)`, so this line was the only one with `shouldNotBeEnabled()`.